moveit_core/CMakeLists.txt

```diff
@@ -52,6 +49,9 @@ find_package(pluginlib REQUIRED)
 # TODO: Port python bindings
 # find_package(pybind11 REQUIRED)
 
+# Finds Boost Components
+include(ConfigExtras.cmake)
+
```

Comment:
It looks like you just moved this line?

Reply:
Sorry, I forgot to revert the previous commit as well. Is that what you were implying was missing?

Reply:
Further up in this file you removed these two lines and then added them back here.  I'm confused about what difference that would make.

Reply:
As per what Jafar said, the first solution involves moving the `include(ConfigExtras.cmake)` line after all the `find_pakage` calls. When we do this, the `include(ConfigExtras.cmake)` will be the last one to call `find_package(Boost ...)`, so `Boost_LIBRARIES` will not be overwritten by any dependencies, allowing us to use ament_* to link.

That's what I did when I removed those two lines from above and added them back there.

Reply:
Thank you for the explination.  I misunderstood what this would do.